core/controllers/profile.py

```diff
@@ -93,7 +95,14 @@ def get(self, username):
             edited_exp_summary_dicts = (
                 summary_services.get_displayable_exp_summary_dicts_matching_ids(
                     user_contributions.edited_exploration_ids))
-        profile_is_of_current_user = (self.username == username)
+            created_collection_summary_dicts = (
+                summary_services.get_displayable_collection_summary_dicts_matching_ids(       #pylint: disable=line-too-long
+                    user_contributions.created_collection_ids))
+            edited_collection_summary_dicts = (
+                summary_services.get_displayable_collection_summary_dicts_matching_ids(       #pylint: disable=line-too-long
+                    user_contributions.edited_collection_ids))
+
+            profile_is_of_current_user = (self.username == username)
```

Comment:
Change to `is_profile_of_current_user`, the current variable implies that the profile is the current user.